views/find.erb

```diff
@@ -0,0 +1,14 @@
+<ul>
```

Comment:
again, I really like that you've separated the views across multiple files for each feature and added HTML styling to it.